.gitmodules

```diff
@@ -16,3 +16,9 @@
 [submodule "minidx9"]
 	path = minidx9
 	url = https://github.com/hrydgard/minidx9.git
+[submodule "common"]
+	path = common
+	url = https://github.com/RPCS3/common
+[submodule "rsx_program_decompiler"]
```

Comment:
Why make the shader recompiler a submodule? Wouldn't it make more sense to be part of the core?